CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# The Odin Project Curriculum Contributing Guide
+
+Thank you for expressing interest in contributing to The Odin Project (TOP) curriculum! Before continuing through this guide, be sure you've read our [general contributing guide](https://github.com/TheOdinProject/.github/blob/ac0adb6bd151dca53798776783c5604d31e3389e/CONTRIBUTING.md), as it contains information that is important for all of our repos.
```

Comment:
Is there a reason this is linking to a specific commit instead of the version on the main branch (ie https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)?

Reply:
Yep!

I copied the url from the wrong tab 😆